packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -466,7 +468,7 @@ export function inferResourceNameFromPath(path: string): string {
     }
   })
 
-  return sanitize(pathNoPathParams)
```

Comment:
We run `sanitize` in `createDataDef`. Before then, all potential names are not sanitized, so sanitizing here (early) seems to be redundant.